### ISSUE TITLE
.

### ISSUE THREAD

**Issue by peer-to-peer-network** [bitcoin/bitcoin#18630]
Yes, I do know what this mean. I actually in less than one day tested successfully something which looks like this: 

WIF: 2H8hoBWyKhcDstTSUQEzCDEv9ZbWCDhLubaaC21a6fxqZBTisahZHBsyJkrnqP7czKoL1rjKFf4BmdYBT8b75URHNdCASBA
Bitcoin address: **1PZKktjLXjd3hYubpMq9yCYnsVEYcNow85**

This :point_up: is ed25519 and the hashing function in BLAKE3. 
I'm currently working on verification and possibly replacing with BLAKE3 the SHA512 hashing in the ed25519 algorithm. (not yet) 

Basically, this is going to be the safest cryptocurrency.

Cracking time required assuming one hundred trillion guesses per second:

Bitcoin = 3.53 thousand trillion trillion trillion trillion centuries
NewBitcoin = 3.53 hundred million trillion trillion trillion trillion trillion trillion trillion trillion centuries

Bitcoin: 1.11 x 10^75
NewBitcoin: 1.11 x 10^128

If anyone want to contribute this this please contact me at **satoshinakamoto@gmx.ch**
